rtl/include/eclhelper.hpp

```diff
@@ -702,6 +702,7 @@ interface ICodeContext : public IResourceContext
     virtual char *getClusterName() = 0; // caller frees return string.
     virtual char *getEnv(const char *name, const char *defaultValue) const = 0;
     virtual char *getGroupName() = 0; // caller frees return string.
+    virtual char *getClusterGroupNames() = 0; // caller frees return string.
```

Comment:
This is going to break previously compiled queries

Reply:
hm, what is the normal approach?
If I move the method to the end of the interface, that will keep existing methods at same offsets?
Or do I need to up the interface version somewhere (ACTIVITY_INTERFACE_VERSION?) and/or is that enough?
@richardkchapman

Reply:
Ah, ok I didn't realize this.  So can this only go into a breaking version ?

Reply:
I think if this is added to the end of the interface it should not break existing queries, and you don't need to increase the version number.